src/components/ResumeUploadBtn.js

```diff
@@ -0,0 +1,41 @@
+import React, { useRef, useState } from 'react'
+import { Button } from '../components/Button'
+import { QuestionHeading } from '../components/Typography'
+
+const resumeStyle = {
+  display: 'flex',
+  alignItems: 'center'
+};
+
+const ResumeUploadBtn = () => {
+  const [hint, setHint] = useState(null)
+  const inputFile = useRef(null);
+
+  const handleClick = () => {
+    inputFile.current.click();
+  }
+
+  const handleChange = (e) => {
```

Comment:
Maybe we shold make this function call a callback function? We are using the container pattern in this repo, aka display components shouldn't handle logic, so we'd probably have a wrapper container handle the file once it's added.

Reply:
thanks for the comments! resolved in latest commit!